File: coco/src/main/java/coco/ep/m7/Divd.java
package coco.ep.m7;

import coco.ep.Exp;

public interface Divd<FT> extends coco.ep.m6.Divd<FT>, coco.ep.m7.Exp<FT> {

    default Exp<FT> powby(Exp<FT> other) {
        return this.divd(convert(getLeft()).powby(convert(other)), convert(getRight()).powby(convert(other)));
    }
}

File: coco_csharp/CoCo/EP/M7/Divd.cs
namespace EP.M7 {
    public interface Divd<FT> : Exp<FT>, EP.M6.Divd<FT> {
        EP.Exp<FT> Exp<FT>.powby(EP.Exp<FT> other) {
            return divd(convert(getLeft()).powby(convert(other)), convert(getRight()).powby(convert(other)));
        }
    }
}